src/control/layer/LayerController.cpp

```diff
@@ -10,8 +10,8 @@
 #include "util/cpp14memory.h"
 
 LayerController::LayerController(Control* control)
- : control(control),
-   selectedPage(size_t_npos)
+ : control(control)
+ , selectedPage(npos)
```

Comment:
The Comma should always be at the end of the line, not at the beginning.

Reply:
I know, but there is a bug in clang-format, so we decided to use this version until the bug is fixed, because it looks more similar to the old style. I could use /clang-format off

Reply:
Ok

Reply:
I think using `//clang-format off ... on` isn't the best idea, because no one will ever find and remove it. We should just wait for clang-9 or 10 fixing this issue.